third_party/move/move-model/src/metadata.rs

```diff
@@ -291,7 +291,7 @@ impl LanguageVersion {
             LanguageVersion::V1 => VERSION_DEFAULT,
             LanguageVersion::V2_0 => VERSION_DEFAULT_LANG_V2,
             LanguageVersion::V2_1 => VERSION_DEFAULT_LANG_V2,
```

Comment:
nit: can combine these for `VERSION_DEFAULT_LANG_V2` into a single case via `
`